applets/002-save-sysctls/applet.py

```diff
@@ -1,21 +1,22 @@
 
 def main(x, conf, args):
     if not len(args) > 0:
-        x.p.msg("no hostname argument given, like alpha\n")
+        x.p.err("no hostname argument given, like alpha\n")
         return False
 
     hostname = args[0]
     ip = conf.get_control_ip(hostname)
     user = conf.get_user(hostname)
     # check if file exist already, if we break here
     # the following four lines are probably obsolete
-    _, _, exit_code = x.ssh.exec(ip, user, "test -f /tmp/sysctl.save")
+    exit_code = x.ssh.exec(ip, user, "test -f /tmp/sysctl.save")
```

Comment:
Ah, yes, i differentiated between "exec" and "exec_verbose". "exec_verbose" now returns the known three-tuple. "exec" just returns the exit code, and uses stdout and stderr suppression. This seemed to be a nice way of handling the output.